### PR TITLE
Minimal .gitignore

### PR DESCRIPTION
**Pull request by mauroservienti**
Should replace https://github.com/Particular/Workshop/pull/251

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,160 +1,19 @@
...
-*.vsp
-*.vspx
-
-# Guidance Automation Toolkit
-*.gpState
+# Nuget packages
+packages/
 
 # ReSharper is a .NET coding add-in
+
```
Comment (adamralph) [inline .gitignore]: Why has this line been added?

**Comment by mauroservienti** [comment]
I marked it as wip so to decide if we wanna go with this or #251 , this is it.

File: .gitignore (hunk 1 above)
Comment (mauroservienti) [reply .gitignore]: It happened to me that sometimes R# detritus are included. I’ve not been using R# for a long time now, but there are people in the company that do use it. Are you using it @adamralph ?

Comment (adamralph) [reply .gitignore]: Sorry, I didn't make myself clear. I just meant the extra blank line after the comment. The other sections don't have it.

No, I don't use R#. 😉

Comment (mauroservienti) [reply .gitignore]: A typo

Comment (mauroservienti) [reply .gitignore]: Fixed

**Comment by dvdstelt** [comment]
Personal question: Why would we want a minimalistic .gitignore file? Is there any benefit?

Devil's Advocate: There should be an issue for this, but there is none. Not even a description?

**Comment by adamralph** [comment]
The benefit is simplicity and removing YAGNI cruft.

I'm not sure we need an issue for such a trivial change.

**Comment by dvdstelt** [comment]
> The benefit is simplicity

How could it be more simple than let it be like Visual Studio ot Git created it?

> removing YAGNI cruft

It's YAGNI to remove it!

**Comment by mauroservienti** [comment]
The one created by default by GitHub or Visual Studio contains all possibile exclusions for all possible languages, including things like C++ or ClickOnce deployments. Whenever we need to manage it it takes more time to understand what’s in there then to edit it.

**Pull request merged**